test/ProblemDetailsMiddlewareFactoryTest.php

```diff
@@ -13,39 +13,50 @@
 use Mezzio\ProblemDetails\ProblemDetailsMiddleware;
 use Mezzio\ProblemDetails\ProblemDetailsMiddlewareFactory;
 use Mezzio\ProblemDetails\ProblemDetailsResponseFactory;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
 use Psr\Container\ContainerInterface;
+use ReflectionObject;
 use RuntimeException;
 
 class ProblemDetailsMiddlewareFactoryTest extends TestCase
 {
-    protected function setUp() : void
+    /** @var ContainerInterface|MockObject */
```

Comment:
Not important for now, but in future, use `&` instead of  `|` in mock types, since a mock implements both interfaces at the same time.

This will pop up in static analysis :-)